documentation/reference/revision-resource-compatibility.md

```diff
@@ -0,0 +1,21 @@
+# Compatible Charm Revisions and Resources
+
+Deploying charms from the `latest/stable` channel *should* guarantee 
+compatibility between charm revision and resource.
+
+However, if you are deploying earlier revisions of the charm please
+use the below table as a reference for compatibility.
+
+
+| Revision | Resource revision | Resource Version |
+|----------|-------------------|------------------|
+| 34       | 18                | 3.1.3            |
+| 33       | 18                | 3.1.3            |
+| 32       | 15                | 3.1.3            |
+| 31       | 14                | 2.1.0            |
+
+A revision and resource version can be specified in the deployment:
```

Comment:
```suggestion
You can specify a revision and resource version can in the deployment as follows:
```
In how-to's we tend to use second person for actions the user can do.

Reply:
Got it thank you!